### ISSUE TITLE
KIF tapOnViewWithAccessibilityLabel doesn't work on iOS9 devices

### ISSUE THREAD

**Issue by hlukman** [kif-framework/KIF#697]
Our KIF tests works in iOS8.4, both on iPhone and iPad, using Xcode 7.   However when we ran the tests on iOS9, they all fail on the very first attempt to tapOnViewWithAccessibilityLabel.  We also tried this with KIF 3.2.3 and got same results.  Screenshot is attached below.

<img width="1383" alt="kif-ios9-fail" src="https://cloud.githubusercontent.com/assets/4713135/9011220/d2e2a2f6-3779-11e5-80c3-ade62e0c3e8c.png">

**Comment by phatmann** [comment]
KIF is not yet compatible with iOS 9. But we are [working on it](https://github.com/kif-framework/KIF/pull/689).
